packages/discord-bot/src/handlers/activate.ts

```diff
@@ -24,21 +24,32 @@ export const activationHandler = async (
     await interaction.reply('Unable to create user account.');
     return;
   }
-  const baseURL = process.env.FRONTEND_URL;
-  if (!baseURL || baseURL === undefined) {
+  const server = process.env.SERVER;
+  const port = process.env.FRONTEND_DEVELOPMENT_PORT;
+
+  if (!server || server === undefined) {
+    await interaction.reply(
+      'ERROR: `SERVER` not defined in environment variables. Contact praise admin'
+    );
+    return;
+  }
+  if (!port || port === undefined) {
     await interaction.reply(
```

Comment:
The checks for env variables are no longer necessary due to #267

Reply:
Gone.